docs/data-sources/cloud_observability_aws_account.md

```diff
@@ -0,0 +1,38 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_cloud_observability_aws_account Data Source - terraform-provider-grafana"
+subcategory: "Cloud Observability"
+description: |-
+  
+---
+
+# grafana_cloud_observability_aws_account (Data Source)
+
+
+
+## Example Usage
+
+```terraform
+data "grafana_cloud_stack" "test" {
+  slug = grafana_cloud_stack.test.slug
+}
+
+data "grafana_cloud_observability_aws_account" "test" {
+  stack_id = data.grafana_cloud_stack.test.id
+  name     = "my-aws-account"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) A name to help identify this AWS Account resource. This name must be unique per StackID. Part of the Terraform Resource ID.
+- `stack_id` (String) The StackID of the Grafana Cloud instance. Part of the Terraform Resource ID.
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `regions` (List of String) A list of regions that this AWS Account resource applies to.
+- `role_arns` (Map of String) A map consisting of one or more name => IAM Role ARN pairs to associate with this AWS Account resource.
```

Comment:
this is out of date right? Since we are going to have a single ARN per account entity

Reply:
it's not out of date yet, Kyle mentioned I should change it in https://github.com/grafana/terraform-provider-grafana/pull/1597#discussion_r1622292945, but I have not implemented the change, yet